pkg/apis/devops/v1alpha3/pipeline_types.go

```diff
@@ -77,6 +77,15 @@ const (
 	MultiBranchPipelineType = "multi-branch-pipeline"
 )
 
+const (
+	SourceType_SVN       = "svn"
+	SourceType_Git       = "git"
+	SourceType_SingleSVN = "single_svn"
+	SourceType_Gitlab    = "gitlab"
+	SourceType_Github    = "github"
+	SourceType_Bitbucket = "bitbucket_server"
+)
+
```

Comment:
We usually use Camel-Case. Just like,

```
SSHAuthUsernameKey
SSHAuthPassphraseKey
```

Reply:
Of course, I agree with you. But I didn't change it in order to keep it compitable. It means I just put them into a const instead of giving a new one.